third-party/chpl-venv/chpldoc-sphinx-project/source/conf.py

```diff
@@ -60,9 +60,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = '0.0.1'
+version = os.environ.get('CHPLDOC_PROJECT_VERSION', '')
 # The full version, including alpha/beta/rc tags.
-release = '0.0.1'
+release = os.environ.get('CHPLDOC_PROJECT_VERSION', '')
```

Comment:
Based on the comments, I would expect these to do different things if an alpha/beta/rc tag was provided, but it doesn't look like they will.

Reply:
Yeah, I got it. Maybe I can trim the version to short at variable 'version'. That will fix it I guess.

Reply:
I think you'll probably want to do some validation anyways to ensure it's in an expected version format

Reply:
what all type of versions can we add? maybe like '*.*.*' or '*.*.*-alpha' or '*.*.*-beta' or '*.*.*-rc'?

Reply:
That seems reasonable to me, but I'd look for other Sphinx project examples online to be certain.  We don't need to support every style necessarily (at least not right away) but focusing on the most standard one seems reasonable

Reply:
I have added validation for the formats which I have specified earlier.